src/main/java/edu/njit/jerse/ashe/services/SpeciminTool.java

```diff
@@ -69,13 +74,43 @@ public static String runSpeciminTool(String root, String targetFile, String targ
     }
 
     /**
-     * Creates a temporary directory for storing output from the Specimin tool.
+     * Deletes all files and subdirectories in the Specimin temporary directory. This method is specifically
+     * important when automating {@link Ashe} with {@link AsheAutomation} and {@link RepositoryAutomationEngine}.
+     * If the temporary directory is not deleted, many directories will be created and require a lot of
+     * disk space.
      *
-     * @return Path of the created temporary directory.
+     * @param tempDirPath the {@link Path} to the directory that needs to be deleted
+     * @throws IOException if an I/O error occurs during file operations
+     */
+    public static void deleteSpeciminTempDir(Path tempDirPath) throws IOException {
```

Comment:
It really stinks that Java doesn't provide a method for deleting a directory.
Rather than re-inventing the wheel, how about using `deleteDir()` from https://plumelib.org/plume-util/api/org/plumelib/util/FilesPlume.html or `FileUtils.deleteDirectory()` from commons-io?

Reply:
Just using deleteDir() within Ashe#run is much better than creating this method. The change has been made and SpeciminTool.java no longer holds this method. Additionally, we actually do not need createTempDirectory() either and just added noise. That too was deleted.